File: evergreentrick.c
#include <stdio.h>
#include <string.h>

int main(){

    char frase1[100];
    char frase2[100];

    scanf ("%s", frase1);
    scanf ("%s", frase2);

    int tam= strlen(frase1) + strlen(frase2);

    char StringFinal[200]= "";


    for (int i=0, j=0, k=0;i<tam;i++, j+=2, k+=2){

        if (j<strlen(frase1)){

            StringFinal[i]+=frase1[j];
            StringFinal[i+1]+=frase1[j+1];
        }

        if (k<strlen(frase2)){

            StringFinal[i+2]+= frase2[k];
            StringFinal[i+3]+= frase2[k+1];
        }
    }

    printf ("%s", StringFinal);
    return 0;
}

File: evergreentrick.java
import java.util.Scanner;


public class evergreentrick{

    public static void main(String[] args) {
        
    
    Scanner scanner= new Scanner (System.in);

    String frase1= scanner.nextLine();
    String frase2= scanner.nextLine();


    int tam= frase1.length() + frase2.length();

        StringBuilder str= new StringBuilder();


    for (int i=0, j=0, z=0; i<tam;i++, j+=2, z+=2){

      if (j<frase1.length()){

        str.append(frase1.charAt(j));
        str.append(frase1.charAt(j+1));

        if (z<frase2.length()){
        str.append(frase2.charAt(z));
        str.append(frase2.charAt(z+1));
        }
      }

    }

    System.out.println (str);

    }
}